From d2cdb468fce5e31e5150c3158871247f866356e0 Mon Sep 17 00:00:00 2001
From: Bruno Braga <brunobraga@google.com>
Date: Fri, 1 Jul 2022 09:07:46 +0000
Subject: [PATCH 11/12] [APC] Remove APC per country conditional delivery.

Binary size reduction not possible since we are removing country restricted packages in order to be served globally.

Binary-Size: Size increase is unavoidable (see above).
Bug: 1335389
Change-Id: Ibfc2f1616e91217db03bdd8bbbf7b868fc0f736e
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3727156
Reviewed-by: Clark DuVall <cduvall@chromium.org>
Reviewed-by: Andrew Grieve <agrieve@chromium.org>
Commit-Queue: Bruno Braga <brunobraga@google.com>
Reviewed-by: Jan Keitel <jkeitel@google.com>
Cr-Commit-Position: refs/heads/main@{#1019996}
---
 ...utofill_assistant.AndroidManifest.expected | 20 ++++++++-----------
 .../java/AndroidManifest.xml                  |  4 ----
 2 files changed, 8 insertions(+), 16 deletions(-)

diff --git a/chrome/android/expectations/monochrome_public_bundle__autofill_assistant.AndroidManifest.expected b/chrome/android/expectations/monochrome_public_bundle__autofill_assistant.AndroidManifest.expected
index 8b0de027cf6f5..08d1809fbc231 100644
--- a/chrome/android/expectations/monochrome_public_bundle__autofill_assistant.AndroidManifest.expected
+++ b/chrome/android/expectations/monochrome_public_bundle__autofill_assistant.AndroidManifest.expected
@@ -6,21 +6,17 @@
     package="org.chromium.chrome.stable"
     platformBuildVersionCode="31"
     platformBuildVersionName="12">
-  <dist:module dist:title="@string/autofill_assistant_module_title">  # DIFF-ANCHOR: afa8402d
-    <dist:delivery>  # DIFF-ANCHOR: f7518c58
-      <dist:install-time>  # DIFF-ANCHOR: 4387071b
-        <dist:conditions>  # DIFF-ANCHOR: 210b2aff
+  <dist:module dist:title="@string/autofill_assistant_module_title">  # DIFF-ANCHOR: 328f25a9
+    <dist:delivery>  # DIFF-ANCHOR: 62b39bbe
+      <dist:install-time>  # DIFF-ANCHOR: a4752ed3
+        <dist:conditions>  # DIFF-ANCHOR: bedacbe5
           <dist:min-sdk dist:value="26"/>
-          <dist:user-countries dist:include="true">  # DIFF-ANCHOR: 285b35e4
-            <dist:country dist:code="GB"/>
-            <dist:country dist:code="US"/>
-          </dist:user-countries>  # DIFF-ANCHOR: 285b35e4
-        </dist:conditions>  # DIFF-ANCHOR: 210b2aff
-      </dist:install-time>  # DIFF-ANCHOR: 4387071b
+        </dist:conditions>  # DIFF-ANCHOR: bedacbe5
+      </dist:install-time>  # DIFF-ANCHOR: a4752ed3
       <dist:on-demand/>
-    </dist:delivery>  # DIFF-ANCHOR: f7518c58
+    </dist:delivery>  # DIFF-ANCHOR: 62b39bbe
     <dist:fusing dist:include="true"/>
-  </dist:module>  # DIFF-ANCHOR: afa8402d
+  </dist:module>  # DIFF-ANCHOR: 328f25a9
   <uses-sdk android:minSdkVersion="24" android:targetSdkVersion="31"/>
   <uses-split android:name="chrome"/>
   <application/>
diff --git a/chrome/android/features/autofill_assistant/java/AndroidManifest.xml b/chrome/android/features/autofill_assistant/java/AndroidManifest.xml
index 395b4ff94407d..17f671531a7a3 100644
--- a/chrome/android/features/autofill_assistant/java/AndroidManifest.xml
+++ b/chrome/android/features/autofill_assistant/java/AndroidManifest.xml
@@ -10,10 +10,6 @@
           <dist:install-time>
             <dist:conditions>
               <dist:min-sdk dist:value="26"/>
-              <dist:user-countries dist:include="true">
-                <dist:country dist:code="US"/>
-                <dist:country dist:code="GB"/>
-              </dist:user-countries>
             </dist:conditions>
           </dist:install-time>
         </dist:delivery>
-- 
2.37.2

